src/client/tile.cpp

```diff
@@ -1330,12 +1330,10 @@ bool TextureSource::generateImagePart(std::string part_of_name,
 				baseimg->fill(video::SColor(0,0,0,0));
 			}
 			while (!sf.at_end()) {
```

Comment:
there's another edge case above here that probably nobody actually uses:
combine inherits the parent dimensions in which case it's totally "valid" to set them to zero or 99999
code should be
```cpp
			u32 h0 = stoi(sf.next(":"));
			if (!baseimg) {
				CHECK_DIM(w0, h0);
				baseimg = driver->createImage(video::ECF_A8R8G8B8, {w0, h0});
				baseimg->fill(video::SColor(0,0,0,0));
			}
```